### PR TITLE
docs: Add documentation about the different file types and their functionalities

### PR DESCRIPTION
**Pull request by flavens**
resolves DEV-1200 and DEV-1272

### CODE REVIEW

File: docs/user-guide/data.md [hunk 1]
```diff
@@ -183,15 +183,61 @@ If you don't have the permissions you were supposed to get, please contact the [
...
+
+##### Text file
+
+Supported format: **csv**, **txt**, **xml**
+
+![Text viewer](../assets/images/text-doc-file.png)
+
+##### Video file
+
+Supported format: **mp4**
```
Comment (mdelez) [inline docs/user-guide/data.md:232]: "supported file types" instead of "supported format"

File: docs/user-guide/data.md [hunk 2]
```diff
@@ -183,15 +183,61 @@ If you don't have the permissions you were supposed to get, please contact the [
...
+
+![Video viewer](../assets/images/video-file.png)
+
+##### Archive file
+
+Supported format: **zip**, **x-tar**, **gzip**
```
Comment (mdelez) [inline docs/user-guide/data.md:238]: After #808 is merged, the supported **file types** (should change from "supported format") will be zip, tar, gz, gzip, 7z, tgz, z

File: docs/user-guide/data.md [hunk 3]
```diff
@@ -183,15 +183,61 @@ If you don't have the permissions you were supposed to get, please contact the [
...
+
+![Audio viewer](../assets/images/audio-file.png)
+
+##### Image file
+
+Supported format: **jpeg**, **jp2**, **tiff**, **tiff-x**, **png**
+
+In a resource of type "still image", you're able to draw regions on the image and annotate this region. Usually, a still image resource is used for book pages, photographs, postcards, letters, etc.
+
+When you open a resource that contains an image, this one is displayed in a viewer. Several functionalities are accessible from the image viewer, e.g. zoom in/out, copy the IIIF link, replace the image or draw a region on the image.
```
Comment (mdelez) [inline docs/user-guide/data.md:209]: "this one" -> "it"

**Review by mdelez — CHANGES_REQUESTED** [review]

**Comment by flavens** [comment]
@mdelez FYI I added a small change requested by Marion (dev-1272)

File: docs/user-guide/data.md [hunk 4]
```diff
@@ -183,14 +183,14 @@ If you don't have the permissions you were supposed to get, please contact the [
 
 #### Display a resource
 
-DSP-APP offers different resource views for different media types (images, videos, audio, archives, text, and PDF files). You get access to the resources from the list of search results. Depending on the media type, DSP-APP offers different tools to work on the resource.
+DSP-APP offers different resource views for different media types (images, videos, audio, archives, text, and PDF files). You can access to the resources from the list of search results. Depending on the media type, DSP-APP offers different tools to work on the resource.
```
Comment (mdelez) [inline docs/user-guide/data.md:186]: You can access ~~to~~ the resources

**Review by mdelez — APPROVED** [review]

**Pull request merged**